packages/flutter_tools/test/src/fake_process_manager.dart

```diff
@@ -221,33 +219,33 @@ abstract class FakeProcessManager implements ProcessManager {
   @protected
   FakeCommand findCommand(
     List<String> command,
-    String workingDirectory,
-    Map<String, String> environment,
+    String? workingDirectory,
+    Map<String, String>? environment,
     Encoding encoding,
   );
 
   int _pid = 9999;
 
   _FakeProcess _runCommand(
     List<String> command,
-    String workingDirectory,
-    Map<String, String> environment,
+    String? workingDirectory,
+    Map<String, String>? environment,
     Encoding encoding,
   ) {
     _pid += 1;
     final FakeCommand fakeCommand = findCommand(command, workingDirectory, environment, encoding);
     if (fakeCommand.exception != null) {
-      throw fakeCommand.exception;
+      throw fakeCommand.exception!;
     }
     if (fakeCommand.onRun != null) {
-      fakeCommand.onRun();
+      fakeCommand.onRun!();
     }
     return _FakeProcess(
       fakeCommand.exitCode,
       fakeCommand.duration,
       _pid,
       fakeCommand.stderr,
-      fakeCommand.stdin,
+      fakeCommand.stdin ?? IOSink(StreamController<List<int>>().sink),
```

Comment:
Can the default for `stdin` be set in the constructor?

Reply:
Done.